app/views/calls/create.js.erb

```diff
@@ -15,7 +15,7 @@ if (location.pathname == Routes.authenticated_root_path) {
     '<%= escape_javascript(render partial: "dashboards/table_content",
                                   locals: { title: t("dashboard.partial_titles.call_list"),
                                             table_type: "call_list",
-                                            patients: current_user.ordered_patients(current_line),
```

Comment:
Should this be `ordered_patients` or `call_list_patients`?

Reply:
ordered! thank you for catching this. I'm going to add a note that this is basically a private func.

Reply:
actually I think I'm turned around on this? From a skim of the code kinda looks like we should be using `call_list_patients` for everything, and `ordered_patients` is basically a private func. Going to review tomorrow and add a documentation note.